src/main/java/seedu/weme/statistics/LikeDataImpl.java

```diff
@@ -0,0 +1,52 @@
+package seedu.weme.statistics;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableMap;
+
+/**
+ * Like data.
```

Comment:
Should have better java docs for this too, maybe describe that this is the backend implementation of the like engine.

Reply:
Ok, will add in.